### DISCUSSION TITLE
Is it time to stop using `createElement` in JSX new transform now?

### DISCUSSION THREAD

**Discussion by magic-akari** [babel/babel#15509] [Q&A]
https://github.com/babel/babel/blob/1584512fef85844f3ba7d2f5c055267ff0fa8f84/packages/babel-plugin-transform-react-jsx/src/create-plugin.ts#L362-L385

I come from https://github.com/swc-project/swc/issues/6987#issuecomment-1475186680.

I have noticed a certain comment in the source code of Babel.

I would like to inquire whether a specific React version requires such intermediary steps.

Is it time to stop using `createElement` now?

**Comment by nicolo-ribaudo** [comment]
I think we should ask to the React team

@gaearon
